kili/queries/api_key/__init__.py

```diff
@@ -128,29 +117,22 @@ def _query_api_keys(self,
     @typechecked
     def count_api_keys(self, api_key_id: Optional[str] = None, user_id: Optional[str] = None,
                        api_key: Optional[str] = None) -> int:
-        """
-        Count and return the number of api keys with the given constraints
-
-        Parameters
-        ----------
-        api_key_id :
-            The unique id of the api key to retrieve.
-        user_id :
-            Identifier of the user (you can only query your own api keys).
-        api_key :
-            Value of the api key (you can only query your own api keys).
-
-        Returns
-        -------
-        dict
-            A result object which contains the query if it was successful, or an error message else.
-
-        Examples
-        -------
-        >>> kili.count_api_keys(user_id=user_id)
-        3
-        >>> kili.count_api_keys(api_key=api_key)
-        1
+        """Count and return the number of api keys with the given constraints.
+
+        Args:
+            api_key_id: Identifier of the API key to retrieve.
+            user_id: Identifier of the user.
+            api_key: Value of the api key.
+
+        Returns:
+            A result object which contains the query if it was successful,
+                or an error message.
+
+        Examples:
+            >>> kili.count_api_keys(user_id=user_id)
```

Comment:
Sometimes examples are escaped in ```, sometimes not. Which option is best?

Reply:
By default "Examples" section is displayed as a code block.
The cases where a wrap the example in ```is when I want to display a comment above the command line. Example:
```
Examples:
# List all issues of a project and their authors
 >>> kili.issues(project_id=project_id, fields=['author.email'])
```
will be displayed as below because he interpreted the # as a markdown h1
<img width="889" alt="Capture d’écran 2022-05-03 à 10 33 24" src="https://user-images.githubusercontent.com/82399630/166424780-77fc7926-c5fd-4c21-aca5-7375ac345f3b.png">

The solution that I found was to force it into a code block by wrapping it inside a ```

Reply:
I just moves the comment on the same line as the command, this resolves the problem